README.md

```diff
@@ -39,10 +39,14 @@ Currently, planet-node produces and uses storage and settings via
 `appsettings.json` and `PN_` prefixed environment variables. if you want to
 change settings, please edit that files or set environment variables.
 
+in bash
 ```
 $ PN_StorePath="/tmp/planet-node" dotnet run --project PlanetNode
 ```
-
+in PowerShell
+```
+$ $Env:PN_StorePath="/tmp/planet-node"; dotnet run --project PlanetNode
```

Comment:
```suggestion
PS > $Env:PN_StorePath="/tmp/planet-node"; dotnet run --project PlanetNode
```

`$` in start of line indicates bash shell. For PowerShell, `PS > ` seems to be used usally. (e.g., [AWS Tools for PowerShell doc](https://docs.aws.amazon.com/powershell/latest/userguide/specifying-your-aws-credentials.html))